lib/elixir/lib/exception.ex

```diff
@@ -653,12 +662,18 @@ defmodule FunctionClauseError do
 end
 
 defmodule Code.LoadError do
-  defexception [:file, :message]
+  defexception [file: nil, message: nil, description: "could not load file"]
```

Comment:
@lexmag I'm afraid that if we remove description: from where it was originally used, it won't be backward compatible the change. like in https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/exception.ex#L513-L538